### DISCUSSION TITLE
How do GitHubs Discussions work? Let's try it.

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#257] [General]
Ok. I have no idea. Sorry. Here aren't any answers (yet).

I created this discussion so anybody can post anything he/she likes and test how do the different features from Github Discussions work.

So, I think, we can adapt us fast to this platform without messing up most important discussions.

**Comment by EdoFro** [comment]
esto lo estoy escribiendo en español. Quiero ver si Github me ayuda a traducirlo.

Actualmente me da la opción *beta* de traducirlo del inglés al español. quiero saber si lo hace al revés también

**Reply by EdoFro** [reply]
OK. I have an option to translate a post from english to spanish. I wanted to know if it could be done otherwise

**Comment by abc16361** [comment]
Hi, just a test.

A pair of **hand-made** flip-flop sandels.
![flip-flop-sandels](https://user-images.githubusercontent.com/101362122/158360257-c5e8e4e5-ad6f-4590-8745-c6d9445bf354.png)
@fp users
- You like it?
- How many do you want?
- [ ] pick up a pair

> dsdsd dsadsd
`dsd dasd dsd dsa`

[freeplane](https://sourceforge.net/projects/freeplane/)

abc163

**Reply by dpolivaev** [reply]
To ban or not to ban?

**Reply by ldmpub** [reply]
It looks like a "valid" test 😜.
